web/src/components/Toast.tsx

```diff
@@ -28,42 +41,37 @@ function Toast({ title, description, toastAction, toastActionText, toastClose }:
       <ToastPrimitive.Root
         open={open}
         onOpenChange={setOpen}
-        className="radix-state-open:animate-toast-slide-in-right radix-swipe-end:animate-toast-swipe-out radix-state-closed:animate-toast-hide fixed left-1/2 top-16 z-50 w-1/4 self-center rounded-lg  bg-white shadow translate-x-radix-toast-swipe-move-x radix-swipe-cancel:translate-x-0 radix-swipe-cancel:duration-200 radix-swipe-cancel:ease-[ease] dark:bg-gray-900"
+        duration={duration}
+        type="background"
+        className="bg-gray/80 fixed top-16 z-50 m-2 inline-block max-w-max self-center rounded-lg shadow backdrop-blur-sm xs:left-1/2 xs:w-max xs:-translate-x-1/2 xs:transform dark:bg-gray-800/80"
```

Comment:
```suggestion
        className="bg-white/80 fixed top-16 z-50 m-2 inline-block max-w-max self-center rounded-lg shadow backdrop-blur-sm xs:left-1/2 xs:w-max xs:-translate-x-1/2 xs:transform dark:bg-gray-800/80"
```
Looks better white white background in my opinion and it matches our map buttons :)

**Current**
![Screenshot 2024-06-20 at 11 40 43](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/6cbb9c4d-9d19-4aed-910e-e4f308178030)

**Proposed**
![Screenshot 2024-06-20 at 11 42 41](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/7e88f255-f078-4ab3-920a-638d589f369b)

Reply:
I thought I had gotten it right the first time but thanks! And I agree, much better!